wzk/ui/grid/Filter.coffee

```diff
@@ -52,6 +52,9 @@ class wzk.ui.grid.Filter extends goog.events.EventTarget
   setValue: (val) ->
     goog.dom.forms.setValue @el, val
 
+  reset: ->
+    goog.dom.forms.setValue @el, ''
```

Comment:
Because I needed to reset all filters when some filtered columns are hidden.
I thought this is a proper way of reseting value of an input / select